### DISCUSSION TITLE
Trigger Words for LORAs

### DISCUSSION THREAD

**Discussion by Rogala** [lllyasviel/Fooocus#2601] [Q&A] [answered]
I am downloading Lora from https://civitai.com/
Some of them have a word or phrases (Trigger Words) for LORA that you type into the promt line for them to work.
The question is does the focus not pick them up as part of the prompt, e.g. wildcards are screened `__wildcards__`, weight is also in (weight: 0.1)?

**Comment by mashb1t** [comment, marked as answer]
Fooocus does indeed pick up those trigger words.

**Reply by Rogala** [reply]
I don’t know if I used focus trigger words as part of the promt or just turned on the LORA itself, that’s the question, maybe it’s worth somehow escaping the trigger word

**Reply by mashb1t** [reply]
trigger words are by design part of the prompt in SD based generation tools, as this is what LoRAs "listen" for.
